website/pages/docs/nia/installation/configure.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: Configure Consul-Terraform-Sync
+sidebar_title: Configure
+description: >-
+  A high level guide to configure Consul-Terraform-Sync.
+---
+
+# Configure Consul-Terraform-Sync
+
+The page will cover the main components for configuring your Network Infrastructure Automation with Consul at a high level. For the full list of configuration options, visit the [Consul-Terraform-Sync Configuration page](/docs/nia/configuration).
+
+## Tasks
+
+A task captures a network automation process by defining which network resources to update for a set of services as those services change over time. Configure Consul-Terraform-Sync with one or more tasks that contain a list of Consul services, a Terraform module, and various Terraform providers.
+
+Within the [`task` block](/docs/nia/configuration#task), the list of services for a task represents the service layer that drives network automation. The `source` is the discovery location of the Terraform module that defines the network automation process for the task.
+
+Review the Terraform module to be used for network automation and identify the Terraform providers required by the module. If the module depends on a set of providers, include the list of provider names in the `providers` field to associate the corresponding provider configuration with the task. These providers will need to be configured later in a separate block.
+
+```hcl
+task {
+  name = "website-x"
+  description = "automate services for website-x"
+  source = "namespace/example/module"
+  version = "1.0.0"
+  providers = ["myprovider"]
+  services = ["web", "api"]
+}
+```
+
+## Terraform Providers
+
+Configuring Terraform providers within Consul-Terraform-Sync requires 2 config components. The first is the [`terraform_provider` block](/docs/nia/configuration#terraform-provider). This block resembles [provider blocks for Terraform configuration](https://www.terraform.io/docs/configuration/providers.html) and has the same responsibility for understanding API interactions and exposing resources for a specific infrastructure platform.
+
+Terraform modules configured for task automation may require configuring the referenced providers. For example, to configure the host address and authentication to interface with your network infrastructure. Refer to the Terraform provider documentation hosted on the [Terraform Registry](https://registry.terraform.io/browse/providers) to find available options. The `terraform_provider` block is loaded by Consul-Terraform-Sync during runtime and processed to be included in [autogenerated Terraform configuration files](/docs/nia/network-drivers#provider) used for task automation. Omitting the `terraform_provider` block for a provider will defer to the Terraform behavior assuming an empty default configuration.
```

Comment:
> For example, to configure the host address and authentication to interface with your network infrastructure.

This reads like a phrase to me. Is there more maybe? e.g. "to configure..., you will need to do x"

Reply:
Ah true true, I don't have the "X" in mind, so maybe I'll simplify it

> For example, configuring the host address and authentication to interface with your network infrastructure.

Reply:
Yes! Gotcha. Using 'configuring' parallels with the previous sentence so I get what the example is related to.